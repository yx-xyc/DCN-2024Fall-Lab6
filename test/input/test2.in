ACTIVE
SYNACK
SDATA
RDATA
CLOSE
ACK
FIN
TIMEOUT