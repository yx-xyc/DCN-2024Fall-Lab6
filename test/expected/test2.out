Event ACTIVE received, current State is CLOSED
Event SYNACK received, current State is SYN_SENT
DATA sent 1
DATA received 1
Event CLOSE received, current State is ESTABLISHED
Event ACK received, current State is FIN_WAIT_1
Event FIN received, current State is FIN_WAIT_2
Event TIMEOUT received, current State is TIME_WAIT
